src/test/java/es/deusto/spq/server/data/dto/AssemblerTest.java

```diff
@@ -226,12 +226,10 @@ public void assembleReview_Review_Assemble() {
                 "testreview",
                 "opinion here",
                 7,
-                timestamp,
-                assembler.assembleHotel(hotel),
-                assembler.assembleUser(guest)
+                timestamp
         );
 
-        Assert.assertEquals(reviewDTO, assembler.assembleReview(review));
```

Comment:
Why this change?

Reply:
yeah sorry i will put it back as it was, it is becaus ei changed the dto and at first i thought i should change that also but then i realized that wasnt the case.